distributed/hdfs.py

```diff
@@ -46,6 +49,8 @@ def read_bytes(fn, executor=None, hdfs=None, lazy=False, delimiter=None, **hdfs_
     blocks = get_block_locations(hdfs, fn)
     filenames = [d['filename'] for d in blocks]
     offsets = [d['offset'] for d in blocks]
+    if not_zero:
+        offsets = [max([o, 1]) for o in offsets]
```

Comment:
What is this for?  If this is avro specific then can we handle it in that function?  Perhaps by submitting a function `lambda b: b.split(delimiter, 1)[1]` onto the relevant future.